mosviz/loaders/jwst_loaders.py

```diff
@@ -59,6 +62,29 @@ def nirspec_spectrum2d_reader(file_name):
     return data
 
 
+@mosviz_level2_loader('NIRSpec 2D Level 2 Spectra')
+def nirspec_level2_reader(file_name):
+    """
+    Data Loader for level2 products.
+    Uses extension information to index
+    fits hdu list. The ext info is included
+    in the file_name as follows: <file_path>[<ext>]
+    """
+    file_name, ext = split_file_name(file_name)
+
+    hdulist = fits.open(file_name)
+
+    data = Data(label="2D Spectra")
+    data.header = hdulist[ext].header
+    data.coords = coordinates_from_header(hdulist[ext].header)
+    data.add_component(hdulist[ext].data, 'Flux')
+    data.add_component(np.sqrt(hdulist[ext + 2].data), 'Uncertainty')
```

Comment:
Do we need to add a `TODO` here about potentially using real uncertainty data in the future?

Reply:
Not sure here. I am expecting these to change once we get proper NIRSpec data, these extensions are placeholders for now.

Reply:
It would be useful to have a comment in the code to that effect.